doc/api/plotting/trame.rst

```diff
@@ -2,21 +2,16 @@
 
 Trame
 -----
-The PyVista :mod:`pyvista.trame` module allows users to access the `Trame
+The PyVista ``pyvista.trame`` module allows users to access the `Trame
 <https://kitware.github.io/trame/index.html>`_ widget from PyVista to create
 web-based 3D visualizations. This allows you to access the VTK pipeline using
 the PyVista API so you can pair PyVista and Trame so that PyVista plotters can
 be used in a web context with both server and client-side rendering.
 
 For the full user guide, see :ref:`trame_jupyter`.
 
-.. currentmodule:: pyvista.trame
 
-.. autosummary::
-   :toctree: _autosummary
-
-   launch_server
-   show_trame
-   elegantly_launch
-   get_or_create_viewer
-   plotter_ui
+.. warning::
+   We are having an issue with Sphinx and numpydoc documenting this module
+   because it is not in the top level namespace of the PyVista package.
+   We will fix this later
```

Comment:
@larsoner, this is what I was referring to. This breaks when not having `trame` in the top-level namespace. So I just removed it for now to get a patch release out. Having this would cause the error

Reply:
Sorry to bug you with this... the line between autosummary and numpydoc is blurry to me and thought you might have seen this before

Reply:
No I haven't, and I know `scipy` does not import most of its namespaces (e.g., `scipy.signal`) when you just do `import scipy` so it must be possible to use numpydoc this way. One possibility is that the import fails on the doc build, but it looks like you didn't change the requirements so that doesn't make a lot of sense.

Reply:
Exactly, this sort of namespacing is super common. Banged my head on the wall over this and wasn't sure what to do so I guess we'll just leave this API undocumented for now

Reply:
Thanks for taking a look!